pkgs/data/icons/gruvbox-dark-icons-gtk/default.nix

```diff
@@ -29,6 +29,6 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/jmattheis/gruvbox-dark-gtk";
     license = licenses.gpl3Only;
     platforms = platforms.unix;
-    maintainers = [ maintainers.nomisiv ];
+    maintainers = with maintainers; [ nomisiv bigshaq9999 ];
```

Comment:
Are you a contributor already? O.o

Reply:
You do not need to list yourself as a maintainer if you are merely sending a patch. Modifying a file don't make you "morally accountable" about the future of this package.

Indeed, the git itself already stores on the metadata the names of those who eventually sent patches to the project.

The intent of `meta.maintainers` attribute is (among many other things) to alert the maintainers of this package via the CI system.

Since you are merely sending a patch, there is no need to add yourself as maintainer.
Please remove it.